Assets/VRTK/Scripts/Interactions/VRTK_ControllerActions.cs

```diff
@@ -122,6 +127,25 @@ public virtual void OnControllerModelInvisible(ControllerActionsEventArgs e)
             }
         }
 
+        public GameObject ControllerAlias {
```

Comment:
Rather than have this as a getter variable, would it be better to put this in the `VRTK_DeviceFinder` as a static method?

something like

```
public static GameObejct GetControllerGameObject(SDK_BaseController.ControllerHand hand, GameObject defaultTo)
{
  // Check if hand is set, if not then just return defaultTo
}
```

This way you don't need to copy/paste this method between Controller Events and Controller Actions